airflow/logging_config.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we need this?

Reply:
Nope. but I added it back as it was the only line changed in the original commit. It's a fixup only to revert the main change.